lib/options/uefi-setup-menu.robot

```diff
@@ -59,7 +59,7 @@ Get UEFI Option
         ...    ${option_path[${i}]}
     END
 
-    ${state}=    Get Option State    ${menu}    ${option_path[${path_len}-1]}    ${VALUE}
```

Comment:
@JanPrusinowski Likely we need this to be ported to: https://github.com/Dasharo/open-source-firmware-validation/commits/regression-msi/

Reply:
@JanPrusinowski link?

Reply:
https://github.com/Dasharo/open-source-firmware-validation/commit/904dd1432f23058a795faff172a352c1ac3785a7